packages/stateless/components/NftCard.tsx

```diff
@@ -262,17 +265,37 @@ export const NftCard = forwardRef<HTMLDivElement, NftCardProps>(
               </div>
             )}
 
-            {/* Floor price */}
-            {floorPrice && (
+            {/* Highest offer */}
+            {highestOffer && (
               <div className="space-y-2">
-                <p className="secondary-text">{t('title.floorPrice')}</p>
+                <p className="secondary-text">{t('title.highestOffer')}</p>
+                <div className="body-text font-mono">
```

Comment:
is there supposed to be `space-y-1` in this div or no? just curious because i don't see any other margins